src/main/java/ch/njol/skript/Updater.java

```diff
@@ -344,7 +312,8 @@ public DownloaderTask(Plugin plugin) {
 		}
 
 		private CommandSender sender;
-		
+
+		@SuppressWarnings("ThrowableNotThrown")
```

Comment:
Is this standard Javac warning suppressed? If not, it is unnecessary. Only nullness analysis and $CASES-OMITTED$ get pass here; they are quite absolutely necessary in Skript.

Reply:
What is cases ommited used by btw? The compiler or some debug tool?

Reply:
It is a comment that tells some IDEs and maybe Javac that an enum switch does not handle all cases.

Reply:
Hmm, are you sure this isn't only for the Eclipse IDE? I mean, fine if it is I suppose, but just wondering, I can't find anything on Google